apps/coscrad-frontend/src/components/term-of-the-day/term-of-the-day.container.tsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from 'react';
+import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { useLoadableTermById } from '../../store/slices/resources';
+import { displayLoadableSearchResult } from '../higher-order-components/display-loadable-search-result';
+import { TermDetailFullViewPresenter } from '../resources/terms/term-detail.full-view.presenter';
+import { useDate } from './use-date';
+
+export const TermOfTheDayContainer = (): JSX.Element => {
+    const { termOfTheDayConfig } = useContext(ConfigurableContentContext);
+
+    const getCurrentTermOfTheDay = useDate();
```

Comment:
not quite-
```ts
const getIdOfTodaysTerm = ({day}: MonthAndDay, termOfTheDayConfig: TermOfTheDayConfig): string => {
    const keyToFind = day.toString(); // you should use `1` for the key in the config and not `01` so this will be smoother

    const searchResult = termOfTheDayConfig[keyToFind];

// TODO consider failing more gracefully
    if(isNullOrUndefined(searchResult)){
    throw new Error(`No term specified for today!`)
}

return searchResult; // this will be an ID for the term for today's date
   
}
const date = useDate();

const termId = getIdOfTodaysTerm(date);
```

Reply:
This is updated and resolved.